mkdocs.yml

```diff
@@ -8,23 +8,15 @@ repo_url: "https://github.com/UI-Lovelace-Minimalist/UI"
 repo_name: "UI-Lovelace-Minimalist/UI"
 edit_url: "edit/dev/docs"
 
+# https://pythonrepo.com/repo/lukasgeiter-mkdocs-awesome-pages-plugin-python-documentation
 nav:
   - Home: "index.md"
-  - Setup:
-      - "setup/index.md"
-      - Installation: "setup/install.md"
-      - Configuration: "setup/configuration.md"
-  - Usage:
-      - "usage/index.md"
-      - Cards:
-          - "usage/cards/index.md"
-          - Light: "usage/cards/light.md"
-  - Development:
-      - "development/index.md"
-      - Docs Style Guide: "development/docs_style_guide.md"
+  - "... | setup/**/*.md"
+  - "... | usage/**/*.md"
+  - "... | development/**/*.md"
```

Comment:
This may look tidier here. But now I have to maintain each order in the respective folder separately, if I see it right and have no central place for it anymore?